### DISCUSSION TITLE
Reduce shortcut pollution by shifting windows by 1 window-width

### DISCUSSION THREAD

**Discussion by jay-meister** [rxhanson/Rectangle#565] [Ideas]
I could not work out how to achieve this, and had a search through the discussions but only found this:
Shifting by "1 unit": https://github.com/rxhanson/Rectangle/discussions/521 but was put on hold due to inability to "press and hold".

The feature I would find most helpful would be move or shift my currently sized window 100% to the left or right. That way I don't need to come up with 4 different shortcuts for 4 different positions:

<img width="364" alt="Screenshot 2021-09-06 at 11 35 28" src="https://user-images.githubusercontent.com/12638261/132204351-7057551c-48ec-4a35-903c-47190a73929e.png">

But instead I want to be able to: have 1 shortcut to resize to "left fourth", then a second shortcut to shift it to the right as many times as I please.  I could also re-use this `shift` shortcut if I am working in `thirds`.

The key benefit is that I only really need to remember a few shortcuts whilst retaining all the flexibility that I personally need/use.

Of course shifting up and down would also be useful for some, presumably.

Has this been discussed or perhaps it is even possible already 🙈  ?

Really appreciate this app 🙏

**Comment by rxhanson** [comment]
Configure just the shortcut for first fourth and press the shortcut multiple times to cycle through fourths.

**Comment by jay-meister** [comment]
That does the trick 👌
